config/config-local.yml

```
@@ -33,4 +33,6 @@ assets:
   ordinals:
     host: "http://ord-poc.devnet.babylonchain.io"
     port: 8888
-    timeout: 5000
\ No newline at end of file
+    timeout: 5000
+terms_acceptance:
```

```suggestion
terms_acceptance_logging:
```

Think it's better to be explicit here.